assets/wizards/engagement/views/reader-activation/index.js

```diff
@@ -87,159 +102,180 @@ export default withWizardScreen( () => {
 
 	const emails = Object.values( config.emails || {} );
 
-	if ( false === hasPlugins ) {
-		return (
-			<>
-				<Notice isError>
-					{ __(
-						'Please activate WooCommerce and WooCommerce Subscriptions plugins to use Reader Activation features.',
-						'newspack'
-					) }
-				</Notice>
-				<PluginInstaller
-					isWaiting={ inFlight }
-					plugins={ [ 'woocommerce', 'woocommerce-subscriptions', 'woocommerce-name-your-price' ] }
-					onStatus={ ( { complete } ) => complete && fetchConfig() }
-					withoutFooterButton={ true }
-				/>
-			</>
-		);
-	}
-
 	return (
 		<>
+			<SectionHeader
+				title={ __( 'Reader Activation', 'newspack' ) }
+				description={ () => (
+					<>
+						{ __(
+							'Newspack’s Reader Activation system is a set of features that aim to increase reader loyalty, promote engagement, and drive revenue. ',
+							'newspack'
+						) }
+						<ExternalLink href={ 'https://help.newspack.com' }>
```

Comment:
We may want to sneak a `TODO` in here so we don't forget to change the link to the actual Reader Activation docs 🙂

Reply:
Good call—added in 50c2b70!

Reply:
Thanks Derrick!